Makefile

```diff
@@ -1,10 +1,10 @@
-PYTHON ?= python3.7
+PYTHON ?= python3.11
 VIRTUAL_ENV ?= ./venv
 
 $(VIRTUAL_ENV): setup.cfg setup.py dev-requirements.txt requirements.txt
 	$(PYTHON) -m venv $(VIRTUAL_ENV)
-	$(VIRTUAL_ENV)/bin/python -m pip install -e . -r dev-requirements.txt
-	$(VIRTUAL_ENV)/bin/python -m pip install -e . -r requirements.txt
+	$(VIRTUAL_ENV)/bin/python3.11 -m pip install -e . -r dev-requirements.txt
+	$(VIRTUAL_ENV)/bin/python3.11 -m pip install -e . -r requirements.txt
```

Comment:
Not needed, and you'll need to change it to 3.12 for the next PR as well if you add the version number to the executable name